indico/modules/events/editing/blueprint.py

```diff
@@ -62,12 +62,17 @@
 _bp.add_url_rule('/editing/api/service/disconnect', 'api_service_disconnect', service.RHDisconnectService,
                  methods=('POST',))
 _bp.add_url_rule('/editing/api/service/status', 'api_service_status', service.RHServiceStatus)
+_bp.add_url_rule('/editing/api/editables/files.zip', 'download_archive', management.RHDownloadArchive)
+_bp.add_url_rule('/editing/api/editables/assign', 'assign_editor', management.RHAssignEditor, methods=('POST',))
+_bp.add_url_rule('/editing/api/editables/assign/me', 'assign_myself', management.RHAssignMyselfAsEditor,
+                 methods=('POST',))
+_bp.add_url_rule('/editing/api/editables/unassign', 'unassign_editor', management.RHUnassignEditor, methods=('POST',))
```

Comment:
I think those should be within `/editing/api/<any(paper,slides,poster):type>/...`

Reply:
Ahh, right, we will have separate lists for each type. For some reason I thought we will have a list of all Editables.